app/data_model/session_store.py

```diff
@@ -1,5 +1,5 @@
-from structlog import get_logger
 import simplejson as json
+from structlog import get_logger
```

Comment:
Not used now so this can be removed

Reply:
Ignore that sorry! I thought this was being added and I couldn't see any new log lines so I thought it wasn't being used.